build-npm-module/index290.js

```diff
@@ -0,0 +1,3 @@
+const caseConverter = require('./index');
```

Comment:
Just a note for myself. We'll eventually want to try and change this to `const caseConverter = require('case_converter');` and get this working locally somehow. Maybe a small npm module that's included in this directory?